### DISCUSSION TITLE
Is it possible to set executable and init vim paths with environment variable?

### DISCUSSION THREAD

**Discussion by noidilin** [vscode-neovim/vscode-neovim#2317] [Q&A]
I tried to set the executable path using the environment variable syntax provided by VS Code, but I couldn't make it work, while other settings seem to work without a problem. Can anyone give me a hint about what might be wrong with my settings?

I am using **_window_** device at home and my workplace, and utilizing environment variable to setup path will be convenient to me.

Here is part of my `settings.json`:

```jsonc
{
  // both version work correctly
  "vscode_custom_css.imports": [
    "file:///${userHome}/.config/vscode/main.css",
    "file:///${env:USERPROFILE}/scoop/apps/vscode/current/data/extensions/brandonkirbyson.vscode-animations-2.0.4/dist/updateHandler.js"
  ],

  // error message: [Failed to start nvim] Neovim spawn error: spawn ${userHome}\scoop\shims\nvim.exe ENOENT
  // "vscode-neovim.neovimExecutablePaths.win32": "${userHome}\\scoop\\shims\\nvim.exe",

  // these work fine
  "vscode-neovim.neovimExecutablePaths.win32": "C:\\Users\\test\\scoop\\shims\\nvim.exe",
  "vscode-neovim.neovimInitVimPaths.win32": "C:\\Users\\test\\AppData\\local\\nvim\\init.lua",
}
```

**Comment by little-yzp** [comment]
maybe the variable ${userHome} only take affect in lauch.json and tasks.json.
